src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -158,34 +171,11 @@ export const GithubConfigurationGet = async (req: Request, res: Response, next:
 
 	const gitHubUserClient = await createUserClient(githubToken, jiraHost, { trigger: "github-configuration-get" }, log, gitHubAppId);
 
-	req.log.debug("found github token");
-
-	if (!jiraHost) {
```

Comment:
Why aren't these protected by FF?

Reply:
Because this is some old dead code. We always have jiraHost because it is under jwt-symmetric-middleware